frontend_tests/node_tests/hotkey.js

```diff
@@ -223,7 +223,7 @@ function stubbing(func_name_to_stub, test_function) {
     assert_mapping('D', 'narrow.stream_cycle_forward');
 
     assert_mapping('c', 'compose_actions.start');
-    assert_mapping('C', 'compose_actions.start');
```

Comment:
Instead of removing this line, you'll need to add a test for the C key and make sure it triggers shortcut_notification.